conf/appoptics-init.sh

```diff
@@ -52,7 +52,9 @@ fi
 
 if [ "$APPOPTICS_ENABLE_MYSQL" = "true" ]; then
     mv /opt/appoptics/etc/plugins.d/mysql.yaml.example /opt/appoptics/etc/plugins.d/mysql.yaml
-    sed -i -e "s/root:admin\@tcp(mysql:3306)/$MYSQL_USER:$MYSQL_PASS\@tcp($MYSQL_HOST:$MYSQL_PORT)/" /opt/appoptics/etc/plugins.d/mysql.yaml
+    if [[ -n ${MYSQL_USER} && -n ${MYSQL_HOST} && -n ${MYSQL_PORT} ]]; then
+        sed -i -e "s/root:admin\@tcp(mysql:3306)/$MYSQL_USER:$MYSQL_PASS\@tcp($MYSQL_HOST:$MYSQL_PORT)/" /opt/appoptics/etc/plugins.d/mysql.yaml
+    fi
```

Comment:
Shouldnt we have some log here saying that "hey, you havent provided us with neither a user not a host nor a port - will use the default config"

Reply:
The logs come out quite fast, immediately after this the user if going to see either:

- Mysql connection successful.
or
- Mysql connection invalid.

Reply:
cool - lets get it in then and if we need to make it more explicit we can change it later